src/parser/expression/operator_expression.cpp

```diff
@@ -35,7 +35,8 @@ void OperatorExpression::DeriveReturnValueType() {
     return t1->GetReturnValueType() < t2->GetReturnValueType();
   });
   const auto &type = (*max_type_child)->GetReturnValueType();
-  NOISEPAGE_ASSERT(type <= type::TypeId::DECIMAL, "Invalid operand type in Operator Expression.");
+  NOISEPAGE_ASSERT(type <= type::TypeId::REAL, "Invalid operand type in Operator Expression.");
```

Comment:
I enountered this when i was making changes. should be <= Decimal too

Reply:
I think we punt that type of change until the end-to-end DECIMAL support PR, especially if you've made those changes already. I wouldn't be able to exercise them yet.